include/sound/sof/info.h

```diff
@@ -28,7 +28,7 @@
 
 /* extended data types that can be appended onto end of sof_ipc_fw_ready */
 enum sof_ipc_ext_data {
-	SOF_IPC_EXT_DMA_BUFFER		= 0,
+	SOF_IPC_EXT_UNUSED		= 0,
```

Comment:
Why not just remove this? Otherwise adding the enum numbers in first patch is not needed.

Reply:
We must not have the enum values misaligned. If we'd just remove we could have SOF_IPC_EXT_WINDOW mean 1 on one side and 0 on the other, which is asking for trouble.

Reply:
@kv2019i besides what @paulstelian97 said it's good to have the enum renamed as UNUSED so that we can reuse it next time we need a new type, instead of creating a new one.

Reply:
@paulstelian97 I mean the first patch assigns explicit numbers to each enum, so after that patch, removing SOF_IPC_EXT_DMA_BUFFER won't have an impact, SOF_IPC_EXT_WINDOW continues to be 1 still, right?

If you prefer rename, then just drop the first patch and just rename enum value 0 to UNUSED and we are done

Reply:
if this has never been used by anyone, then it'd be ok to just remove this (keeping other explicit numbers). However, if this was used somewhere, maybe by some older versions, then usually you would add a comment saying, that there used to be an enum field called xxx with value 0. It's been removed, but the value 0 should remain reserved to avoid breaking old software.